src/wallet/rpc/backup.cpp

```diff
@@ -1329,8 +1329,6 @@ RPCHelpMan importmulti()
     // the user could have gotten from another RPC command prior to now
     wallet.BlockUntilSyncedToCurrentChain();
 
-    RPCTypeCheck(mainRequest.params, {UniValue::VARR, UniValue::VOBJ});
-
     EnsureLegacyScriptPubKeyMan(*pwallet, true);
 
     const UniValue& requests = mainRequest.params[0];
```

Comment:
could add:
```suggestion
    const UniValue& requests = mainRequest.params[0].get_array();
```

Reply:
Why? Also, seems unrelated?

Reply:
IIRC, it is because we access `requests` without checking if it's an array or not. The `requests .size()` call and `requests[i]` access do not throw an exception if it is not.

Reply:
> without checking if it's an array or not

Checking the type is the goal of this pull request. However, the type was usually already checked before, so this pull request is a refactor. Are you saying that this pull is accidentally fixing a bug?

Reply:
Not really. We were checking the type here before with the `RPCTypeCheck` that you removed and now we are checking it at the server's dispatcher function in a general manner.

I was just suggesting to unify the code because there was a bug in the past due a missing `get_array` call in other command. E.g. if you provided another type instead of an array, and there was no `RPCTypeCheck` call before it, the usual for loop using `request.size()` and the brackets operator was just skipping the argument, not throwing the invalid type exception.

Still, not a big deal at all because we are now checking the type for all the commands in a general manner anyway. So, feel free to ignore this and the other comment.

Reply:
Anyone is free to change the code in a separate pull request. I can be pinged for review, but I want to keep this pull minimal for now.